spec/openapi.yaml

```diff
@@ -222,37 +119,31 @@ paths:
                 type: object
                 properties:
```

Comment:
suggestion: 
get /api/printer
{ temperature, sd, state, telemetry }

Reply:
It was only subnode of state (state->telemetry). But I changed it as you wanted.